From a08c19ebe366ae5fdd153ecc8990075734bedf24 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Wed, 6 Dec 2023 09:36:20 +0100
Subject: [PATCH 0587/1695] dts: stm32 add tamper pin nodes for stm32mp15x
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These are the pin that can be used on stm32mp15x to use tamper.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 74be18feabb7a9cf4dd04ea9ac07fa2d87ecdc53 dts: stm32 add tamper pin nodes for stm32mp15x]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ica9cbcbb8cf160c073181abe3b608291dda97ea6
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/344288
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp15-pinctrl.dtsi | 37 ++++++++++++++++++++++++
 1 file changed, 37 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp15-pinctrl.dtsi b/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
index 7edff9ca8..34044dd4f 100644
--- a/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
+++ b/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
@@ -1828,6 +1828,43 @@
 		};
 	};
 
+	tamp0_in1_pin_a: tamp0-in1-pin-0 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 13, RSVD)>;
+			st,tamp-id = <1>;
+		};
+	};
+	tamp0_in2_pin_a: tamp0-in2-pin-0 {
+		pins {
+			pinmux = <STM32_PINMUX('I', 8, RSVD)>;
+			st,tamp-id = <2>;
+		};
+	};
+	tamp0_in3_pin_a: tamp0-in3-pin-0 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 1, RSVD)>;
+			st,tamp-id = <3>;
+		};
+	};
+	tamp0_out2_pin_a: tamp0-out2-pin-0 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 13, RSVD)>;
+			st,tamp-id = <2>;
+		};
+	};
+	tamp0_out3_pin_a: tamp0-out3-pin-0 {
+		pins {
+			pinmux = <STM32_PINMUX('I', 8, RSVD)>;
+			st,tamp-id = <3>;
+		};
+	};
+	tamp0_out3_pin_b: tamp0-out3-pin-1 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 13, RSVD)>;
+			st,tamp-id = <3>;
+		};
+	};
+
 	uart4_pins_a: uart4-0 {
 		pins1 {
 			pinmux = <STM32_PINMUX('G', 11, AF6)>; /* UART4_TX */
-- 
2.47.2

